context/app/static/js/components/Detail/ProvVis/tests/ProvData.test.js

```diff
@@ -0,0 +1,171 @@
+import expect from 'expect';
```

Comment:
Rename `tests` directory `__tests__`?

Reply:
We are also using `Name.spec.js ` to name tests elsewhere. I don't have a strong preference on which pattern should be used, but we should be consistent.

Reply:
will do.